src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -2371,6 +2423,7 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
                 <div className={'tabMessageContainer'}>
                     <OqlStatusBanner className="plots-oql-status-banner" store={this.props.store} tabReflectsOql={false} />
                     <AlterationFilterWarning store={this.props.store} isUnaffected={true}/>
+                    {this.quickConfigurations}
```

Comment:
should be inside the plotsTab div, not up here in message container.

Reply:
this will also allow us to tighten up space under the buttons.  should the same as other places we use the pills-nav 
![image](https://user-images.githubusercontent.com/186521/67040725-0f3e3e00-f0f2-11e9-8b0e-08cb312d43ce.png)

Reply:
![Screenshot from 2019-10-17 16-24-20](https://user-images.githubusercontent.com/20069833/67044612-a14a4480-f0fa-11e9-9c77-904a98b79491.png)